lib/components/base-components/Renderable.ts

```diff
@@ -22,6 +22,7 @@ export const orderedRenderPhases = [
   "SchematicPortRender",
   "SchematicLayout",
   "SchematicTraceRender",
+  "BoardAutoSize",
```

Comment:
Can we name this PcbBoardAutoSize

Reply:
I also feel like we should run this stage later, after all the pcb components have rendered